firefox-ios/Client/Application/AppDelegate+SyncSentTabs.swift

```diff
@@ -46,4 +47,14 @@ class AppSendTabDelegate: SendTabDelegate {
             }
         }
     }
+
+    func closeTabs(for urls: [URL]) {
+        mainQueue.async {
+            Task {
+                for url in urls {
+                    await self.applicationHelper.closeTab(url)
+                }
+            }
+        }
```

Comment:
Since this async call is just calling a `Task` which wraps everything else I don't believe it's necessary (the `mainQueue.async` won't have any effect AFAICS), the Task will be executed on a background queue. LMK if I'm missing something here however.

```suggestion
            Task {
                for url in urls {
                    await self.applicationHelper.closeTab(url)
                }
            }
```

Reply:
good catch! I was mainly copying how send tabs does it above this one and figured there was a "good" reason. However I think you're right it's unnecessary in this scenario -- everything seems to work okay in testing!